packages/shared/src/lib/core/nfts/stores/active-profile-nfts-per-account.store.ts

```diff
@@ -36,6 +37,7 @@ export function addOrUpdateNftsForAccount(accountIndex: number, nfts: Nft[]): vo
                 Object.assign(existingNft, nft)
             } else {
                 state[accountIndex].push(nft)
+                void addPersistedCollection(nft)
```

Comment:
i think we want here to keep track of nfts that were added and not updated, and then `addPersistedCollection` for multiple nfts at the same time.

```suggestion
                addedNfts.push(nft)
```